nexus/db-model/src/sled.rs

```diff
@@ -46,6 +46,7 @@ pub struct Sled {
     #[diesel(embed)]
     identity: SledIdentity,
```

Comment:
The "identity" struct already contains a `time_modified` field, fwiw. 

That's why this is currently in the DB, otherwise it would require a schema change: https://github.com/oxidecomputer/omicron/blob/00971809e1326a35f41206413ae87bd0db0d338c/schema/crdb/dbinit.sql#L113-L119

Reply:
I had no idea that was a thing. A little bit too much magic for my personal taste.

Reply:
Changed in [9c6efef](https://github.com/oxidecomputer/omicron/pull/5526/commits/9c6efef90c4c01fc3a74d27dbeb7ae8f13517c0b) to use the `identity` field. I removed the redundant `time_modified`.